packages/strapi-admin/admin/src/components/Roles/Permissions/ContentTypes/ContentTypesRow/index.js

```diff
@@ -209,26 +218,31 @@ const ContentTypeRow = ({ index, contentType, permissionsLayout }) => {
             </CollapseLabel>
           </PermissionName>
           <PermissionWrapper disabled>
-            {permissionsToDisplay.map(permissionLayout =>
-              !isAttributeAction(permissionLayout.action) ? (
-                <PermissionCheckbox
-                  key={permissionLayout.action}
-                  hasConditions={checkConditions(permissionLayout.action)}
-                  disabled
-                  value={hasContentTypeAction(permissionLayout.action)}
-                  name={`${contentType.name}-${permissionLayout.action}`}
-                />
-              ) : (
+            {permissionsToDisplay.map(permissionLayout => {
+              const { action } = permissionLayout;
+              /* eslint-disable */
+              const checkboxProps = isAttributeAction(action)
+                ? {
+                    someChecked: hasSomeAttributeByAction(action),
+                    onChange: () => handleActionSelect(action),
```

Comment:
`handleActionSelect` and `handleContentTypeActionSelect`  are not defined in CE.